### PR TITLE
Improve README for project and generated app

### PR DESCRIPTION
**Pull request by mraible**

### CODE REVIEW

**Comment by mraible** [comment]
@mshima We might want to move the Okta and Auth0 documentation to the project's README and then just link to it from the generated one. I'm not sure what the best approach is.

File: README.md [hunk 1]
```diff
@@ -1,35 +1,21 @@
...
 
-To install this module:
+To install or update this blueprint:
 
 ```bash
-npm install -g generator-jhipster-ionic yo
+npm install -g generator-jhipster-ionic
 ```
 
 To update this module:
```
Comment (mshima) [inline README.md:21]:
```suggestion
To update this blueprint:
```

**Comment by mshima** [comment]
What about adding a section indicating the benefits of OAuth2 to mobile applications and add Okta and Auth0 sections to it?

**Comment by mraible** [comment]
> What about adding a section indicating the benefits of OAuth2 to mobile applications and add Okta and Auth0 sections to it?

Done in https://github.com/mshima/generator-jhipster-ionic/pull/3.

**Pull request merged**